README.md

```diff
@@ -11,9 +11,21 @@ A free and open source manga reading application for iOS and iPadOS.
 - [ ] Update notifications
 
 ## Installation
-The latest ipa will always be available from the [releases page](https://github.com/Aidoku/Aidoku/releases).
 
-For detailed installation instructions, check out [the website](https://aidoku.app). To join the TestFlight, you will need to join the [Aidoku Discord](https://discord.gg/9U8cC5Zk3s).
+For detailed installation instructions, check out [the website](https://aidoku.app).
+
+### Testflight
+
+To join the TestFlight, you will need to join the [Aidoku Discord](https://discord.gg/9U8cC5Zk3s).
+
+### AltStore
+
+- [Add to AltStore](altstore://source/?url=https://raw.githubusercontent.com/Aidoku/Aidoku/altstore/apps.json).
+- Copy [source direct URL](https://raw.githubusercontent.com/Aidoku/Aidoku/altstore/apps.json) and paste to AltStore.
```

Comment:
i don't really care enough to make a redirection page on the website, so i think it's fine just to tell people to copy and paste the link
```suggestion
We have an AltStore repo that contains the latest releases ipa. You can copy the [direct source URL](https://raw.githubusercontent.com/Aidoku/Aidoku/altstore/apps.json) and paste it into AltStore.
```